tcpproxy.go

```diff
@@ -80,6 +80,12 @@ type Proxy struct {
 	ListenFunc func(net, laddr string) (net.Listener, error)
 }
 
+type config struct {
+	routes      []route
+	acmeTargets []Target
+	stopAcme    bool
```

Comment:
what does stopACME mean? when?

I thought acme-stopping would be a special target type so you can SNI-blast two hostnames but not a third.